train_rl4lm.py

```diff
@@ -0,0 +1,143 @@
+import os
```

Comment:
For now this works as a demo / test of rl4lms loading and running with a custom config, from a .yaml file. 
This could be adapted to be a proper test, however: 
- calling the rl4lm api directly inside a pytest doesn't work. 
- this specific script would take quite long to run